crates/cargo-codspeed/Cargo.toml

```diff
@@ -30,6 +30,3 @@ assert_cmd = "2.0.15"
 fs_extra = "1.3.0"
 predicates = "3.1.2"
 uuid = { version = "1.10.0", features = ["v4"] }
-
-[features]
-vendored-openssl = ["cargo/vendored-openssl"]
```

Comment:
If we remove the feature, we need to make a major release. Or have deprecation message